components/button/button-solid.scss

```diff
@@ -1,31 +1,39 @@
 @import "../../lib/reset";
 @import "../../lib/colors";
 @import "../../lib/units";
+@import "../lib/fonts";
 
 
 @mixin button-solid() {
 
     &.mmf-button {
         @include reset-button();
-        background-color: inherit;
-        border: $mmf-input-border--width $mmf-style--border map-get($color-solid, 8);
+        @include solid-font-style();
+        background-color: map-get($color-solid, 6);
+        border: $mmf-input-border--width $mmf-style--border transparent;
         border-radius: $mmf-radius;
         color: map-get($color-solid, 8);
+        padding: 0 $mmf-unit;
 
         &[disabled] {
-            border-color: rgba(map-get($color-solid, 8), map-get($opacity, 2));
             color: rgba(map-get($color-solid, 8), map-get($opacity, 2));
         }
 
         &.is-enabled {
             &:hover,
             &:focus {
-                background-color: map-get($color-solid, 5);
+                background-color: map-get($color-solid, 9);
+                color: map-get($color-solid, 0);
             }
 
             &:focus {
                 @include elevation(1);
             }
+
+            &.activated {
```

Comment:
Whats the difference between is-enabled and activated? This looks misleading.

Reply:
enabled = button is not disabled, hence clickable
activated = button is activated until it is deactivated by clicking it again
Is that understandable? Or shall I rename those classes?